### PR TITLE
Form

### PR DESCRIPTION
**Pull request by F-u-n-H-o-u-s-e**

### CODE REVIEW

**Comment by F-u-n-H-o-u-s-e** [comment]
Form.

File: readme.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
 # HTML form
+
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+
+- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_html-form/report/html_report/)
```
Comment (ArturErmolenko) [inline readme.md:6]: don't forget to add this links in your PR description

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,137 @@
...
+          required>
+        </label>
+
+        <label class = "labels" for="Old">
+          How old are You? 
+          <input autocomplete="off" type="number" 
+          name="Old"
+          value="12" maxlength="3"
+          id="Old" min = "1"
+          class="inputs" max = "100">
```
Comment (ArturErmolenko) [inline src/index.html:35]:
```suggestion
          <input
            autocomplete="off" 
            type="number" 
            name="Old"
            value="12" 
            maxlength="3"
            id="Old" 
            min = "1"
            class="inputs"
            max = "100"
           >
```

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,137 @@
...
+        </label>
+      </fieldset>
+      
+      <fieldset class="groups">
+        <legend>An interesting fact about you!</legend>
+        <label class = "labels inputs" for="Pets1">
+          Do you love cats? 
+          <input autocomplete="off" type="radio" 
+          name="PetsY" > 
+          Yes
```
Comment (ArturErmolenko) [inline src/index.html:81]:
```suggestion
          <input 
            autocomplete="off" 
            type="radio" 
            name="PetsY" 
          > 
            Yes
```

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Check your code formatting(look at Good Example)
formatting

**Comment by F-u-n-H-o-u-s-e** [comment]
Okey, thank you.

**Comment by F-u-n-H-o-u-s-e** [comment]
DEMO LINK
TEST REPORT LINK

**Pull request closed without merging**